packages/opentelemetry-core/src/context/propagation/NoopBinaryFormat.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SpanContext } from '@opentelemetry/types';
+
+/**
+ * No-op implementations of {@link BinaryFormat}.
+ *
+ * @todo: Implement BinaryFormat based on https://github.com/open-telemetry/opentelemetry-js/pull/74
+ */
+class NoopBinaryFormat {
```

Comment:
I know this is needed because it's enforced by the spec, but just wanted to mention for future reference that an `inject` and `extract` methods on the tracer would allow to keep the noop in a single place regardless of the format. No need for additional noop formats for every new format.

Reply:
Agreed, once you have common interface no need for additional noop formats for every new format.